pkg/csi/service/wcp/controller.go

```diff
@@ -2032,11 +2040,13 @@ func (c *controller) CreateSnapshot(ctx context.Context, req *csi.CreateSnapshot
 			}
 			snapshotID, cnsSnapshotInfo, err = common.CreateSnapshotUtil(ctx, c.manager.VolumeManager,
 				volumeID, req.Name, &cnsvolume.CreateSnapshotExtraParams{
-					StorageClassName:           cnsVolumeInfo.Spec.StorageClassName,
-					StoragePolicyID:            cnsVolumeInfo.Spec.StoragePolicyID,
-					Namespace:                  cnsVolumeInfo.Spec.Namespace,
-					Capacity:                   cnsVolumeInfo.Spec.Capacity,
-					IsStorageQuotaM2FSSEnabled: isStorageQuotaM2FSSEnabled,
+					StorageClassName:               cnsVolumeInfo.Spec.StorageClassName,
```

Comment:
req.Name is already the snapshot UUID.

Reply:
done. removed from CreateSnapshotExtraParams as CreateSnapshotUtil already takes req.name as parameter.